test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -117,304 +120,380 @@ void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin)
 		//Select test by name if --singletest is set and not filling state tests as blockchain
 		if (!Options::get().fillchain && !TestOutputHelper::passTest(testname))
 		{
-			o.clear(); //don't add irrelevant tests to the final file when filling
-			continue;
+			if (_fillin)
```

Comment:
`_fillin` and `Options::get().fillchain` look very similar.  Do they always have the same value?  If not, what is the difference between them?